### PR TITLE
Add legal name to user and org creation form

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4752

### User Creation

personal-account-creation

### Organization Creation

org-profile-creation

### CODE REVIEW

File: components/CreateProfile.js [hunk 1]
```diff
@@ -315,6 +361,30 @@ const CreateProfile = ({
...
+              required={false}
+              label={formatMessage(messages.legalName)}
+              mt={3}
+              isPrivate
+              hint={
+                <FormattedMessage
+                  id="legalName.description"
+                  defaultMessage="The legal name is private and shared with the hosts for donation receipts, tax forms and when you submit an expense. This name is not displayed publicly and it must be your legal name."
+                />
+              }
```
Comment (Betree) [inline components/CreateProfile.js:377]: Let's not repeat this, we already describe the field above

image


```suggestion
```

File: components/expenses/ExpenseFormPayeeSignUpStep.js [hunk 2]
```diff
@@ -314,7 +314,13 @@ const ExpenseFormPayeeSignUpStep = ({ formik, collective, onCancel, onNext }) =>
           <Box>
             <Field name={payeeType === PAYEE_TYPE.ORG ? 'payee.organization.legalName' : 'payee.legalName'}>
               {({ field }) => (
-                <StyledInputField name={field.name} label={formatMessage(msg.legalName)} labelFontSize="13px" mt={3}>
+                <StyledInputField
+                  name={field.name}
+                  isPrivate
```
Comment (Betree) [inline components/expenses/ExpenseFormPayeeSignUpStep.js:319]: It seems that we're not requiring it. Should we?

```suggestion
                  isPrivate
                  required={false}
```

**Review by Betree — COMMENTED** [review]
Some questions/comments below. Looks ready to merge

File: components/expenses/ExpenseFormPayeeSignUpStep.js (hunk 2 above)
Comment (SudharakaP) [reply components/expenses/ExpenseFormPayeeSignUpStep.js:315]: Ah yes, this is not required. I've corrected it. 👍🏽

**Review by Betree — APPROVED** [review]

**Pull request merged**